### PR TITLE
Add: Rounding - Square infoboxes

### PR DESCRIPTION
**Pull request by 2641a40fd44383320adde4b027a1d0b03bd550**
**Describe the PR**
adds the `userChrome.rounding.square_infobox` option

**PR Type**

- [x] `Add:` Add feature or enhanced.
- [ ] `Fix:` Bug fix or change default values.
- [ ] `Clean:` Refactoring.
- [ ] `Doc:` Update docs.

**Related Issue**

#645 #618 #616

### CODE REVIEW

**Comment by Patricek0056** [comment]
Thank you, you are doing nice work. To be honest, you are an excellent CSS programmer.

**Comment by 2641a40fd44383320adde4b027a1d0b03bd550** [comment]
I barely did anything

**Review by black7375 — COMMENTED** [review]
IMO, It looks better to add content's infobox into `userChrome.rounding.square_field` and make notification bar(`.container.infobar`) as `userChrome.rounding.square_toolbar`.

I think `userChrome.rounding.square_toolbar` would be good as a name to support [`userChrome.tabbar.one_liner`](https://github.com/black7375/Firefox-UI-Fix/wiki/Options#one-liner) or `userChrome.findbar.floating_on_top` later.

Thank you for contributing!! It's a good suggestion.

**Comment by 2641a40fd44383320adde4b027a1d0b03bd550** [comment]
I don't know, putting content's infobox in field makes no sense to me, as field, for me, is something that takes input from the user

**Comment by Patricek0056** [comment]
I don't want to be rude, but Mr. 2641a40 is right. Putting infobox into field, makes no sense also for me.

**Comment by Patricek0056** [comment]
By the way, this PR fixes also this? 
The warning tooltip in the Clear All History isn't supporting square corners

**Comment by black7375** [comment]
To explain my point of view, I focused on whether the user would use it together rather than the property.
It is not bad to think of it as a forbidden field to write, as if you can't modify the ID of the site after signing up.

For example, the following is a field that looks like infobox but is writable.

image

image

image

**Comment by 2641a40fd44383320adde4b027a1d0b03bd550** [comment]
> By the way, this PR fixes also this?

yes

> the following is a field that looks like infobox but is writable

I see, do I move them to field ?

**Comment by black7375** [comment]
Yes I believe that is better

**Comment by 2641a40fd44383320adde4b027a1d0b03bd550** [comment]
so `.container.infobar` will become `userChrome.rounding.square_toolbar` and others will be left alone, right ?

**Comment by Patricek0056** [comment]
Sorry, but you are talking to me? Or to Mr. black7375?

**Comment by 2641a40fd44383320adde4b027a1d0b03bd550** [comment]
forgot, but @black7375

File: src/contents/_rounding.scss [hunk 1]
```diff
@@ -154,3 +155,29 @@
...
+    regexp "^chrome:\/\/\\w+\/content\/.*\.xhtml$"
+  ) {
+    /*
+     * .container is inside shadow DOM,
+     * and it does not have any other classes,
+     * but there may be other elements with the same class,
+     * hence the weird selector.
+     */
+    link[href="chrome://global/content/elements/message-bar.css"] + .container,
+    .info-box-container:not(.content-blocking-category),
```
Comment (black7375) [inline src/contents/_rounding.scss:171]: This is just a question.
Does Messagebar appear on the web content? Or is it another elements?

Comment (2641a40fd44383320adde4b027a1d0b03bd550) [reply src/contents/_rounding.scss:171]: it will appear in about:addons if you delete an extension

Comment (black7375) [reply src/contents/_rounding.scss:171]: It has been confirmed. Thank you for waiting for a long time.
image

**Comment by Patricek0056** [comment]
And, it is under what option? Under `userChrome.rounding.square_toolbar`, or under `userChrome.rounding.square_infobox`?

**Comment by black7375** [comment]
I'll change it later. (userChrome's notification bar)

**Pull request merged**